src/rrdp.rs

```diff
@@ -1055,7 +1074,7 @@ mod tests {
 
             // Update
             let notification_uri =
-                https("https://krill-ui-dev.do.nlnetlabs.nl/rrdp/notification.xml");
+                https("https://krill-ui-dev.do.nlnetlabs.nl/rrdp/notifyerthingy.xml");
```

Comment:
What is the purpose of setting notification URI differently the second time in this test? It doesn't seem to be verified below other than that nothing fails. Is this trying to verifying the scenario of what happens if between runs krill-sync is invoked with a different notification URI?

Reply:
This is because it's resolving this name to a file on disk which was renamed to support the test done in `process.rs` - where we explicitly changed this from the previously hardcoded 'notification.xml' to something else, and verify that this file is read, and then written to the local synced dir using the same name.